README.md

```diff
@@ -1,4 +1,4 @@
-# ember-m3 [![Build Status](https://secure.travis-ci.org/hjdivad/ember-m3.svg?branch=master)](http://travis-ci.org/hjdivad/ember-m3)
+# Ember-m3 [![Build Status](https://secure.travis-ci.org/hjdivad/ember-m3.svg?branch=master)](http://travis-ci.org/hjdivad/ember-m3)
```

Comment:
Nope I'm convinced; I'll switch everything to `ember-m3` and consider `Ember m3` or `Ember M3` documentation bugs.